src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        String randomColor = colorSupplier.getRandomColor();
+        ball.setColor(randomColor);
+        ball.setNumber(random.nextInt(MAX_VALUE));

```

Comment:
use the constructor for init ball

Reply:
what do you mean? I created one Ball object with default constructor erlier and then used "set" methods

Reply:
create ball and init it using constructor without setters